src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -26,13 +31,14 @@ const styles = StyleSheet.create({
   content: {
     position: "relative",
     padding: size(3),
-    paddingVertical: size(8),
+    paddingVertical: size(5),
```

Comment:
```suggestion
    paddingTop: size(5),
```

Since we are using `paddingBottom` already and they are different values

Reply:
Oooh ok! Will make the changes haha

Reply:
Did it in the commit below~ :3 didnt know can commit the suggestion haha